README.md

```diff
@@ -1,34 +1,72 @@
+# BRL-CAD Web 2022
+
+[![IRC](https://img.shields.io/badge/chat-on%20irc%20channel-green.svg)](http://webchat.freenode.net/?channels=#brlcad)
+[![IRC](https://img.shields.io/badge/join-mailing%20list-green.svg)](https://lists.sourceforge.net/lists/listinfo/brlcad-devel)
+
```

Comment:
You can probably add zulip here as well.

Reply:
Added